### DISCUSSION TITLE
Which one is real? Who should I believe, or did I set it up incorrectly?

### DISCUSSION THREAD

**Discussion by yanite** [winsiderss/systeminformer#2246] [General Discussion]
![image](https://github.com/user-attachments/assets/308784c5-e18c-4fd4-bafd-33d96cf84175)

I am compiling the Linux kernel using WSL, and htop shows one core is fully utilized while the others are idle. Systeminformation also shows the same. However, the Windows built-in Resource Monitor does not show this. Where is the problem?

Everything is the latest version.！

Any suggestions or ideas are welcome!

**Comment by dmex** [comment]
> Where is the problem?

You can't compare apples to oranges because they're different.

System Informer and Resource Monitor are showing `CPU utilization` which is why they're identical but Microsoft removed the `CPU utilization` statistics from Task Manager over 15 years ago and replaced utilization with `Processor Utility` which is a different metric.

It might have been possible to compare with Task Manager were the values somewhat accurate but they're not.  If you generate 50% utilization with a benchmarking tool and compare with how System Informer and everything else are correctly showing 50% utilization while Task Manager is showing wildly inaccurate values like 30% or 70% then it's not accurate and unusable.

Google task manager nonsense and you'll find all the complaints and articles:

* https://aaron-margosis.medium.com/task-managers-cpu-numbers-are-all-but-meaningless-2d165b421e43
* https://illuminati.services/2021/03/17/windows-10-task-manager-cpu-inaccurate-a-tale-of-two-metrics/
